app/main.py

```diff
@@ -43,10 +43,10 @@ def create_smooth_mobile_app(self) -> str:
 
 class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
     def __init__(self, name: str) -> None:
-        super(FullStackDeveloper, self).__init__(name)
+        super().__init__(name)
         self.skills = ["Python", "SQL", "Django", "JavaScript", "HTML", "CSS"]
```

Comment:
your code works correctly and pass all tests. But i'm not sure about logic of this code.
in FullStackDeveloper you just manually wrote skills from parents classes. Probably the main idea of this task to add in child classes those skills which there are in parents classes. Also in checklist there is hint about it

Reply:
yeah, my bad, I forgot to check the checklist. And I thought about code duplicate in my case, but hadn't a good idea how I could solve it